### DISCUSSION TITLE
Please light an IRC channel on Libera and turn off the one on Freenode (if possible)

### DISCUSSION THREAD

**Discussion by davidfetter** [Homebrew/discussions#2127] [Everyday usage]
Folks,

The Freenode IRC channel was a great resource until Freenode itself got captured by hostile actors. I get that it's a bit of a hassle, but it would be really helpful for people who use IRC to have a designated channel to use over on Libera.

**Comment by alebcay** [comment]
Hi, thanks for your concern and bringing this up. As far as I know, none of the maintainer team is on IRC nowadays (see https://github.com/Homebrew/discussions/discussions/86 for similar discussion). There may be a user community on Freenode in the `#homebrew` channel but the maintainer team here has no control on the governance/operations of that channel. To my knowledge, the `#homebrew` channel on Freenode predates the Freenode takeover.

In the same vein, I don't think there's any particular interest (or bandwidth) on the maintainer team right now in being active on Libera (or on IRC in general). As has happened on Freenode in the past, you're of course welcome to start your own discussions/communities on Libera to discuss Homebrew if you wish.

**Comment by gwerbin** [comment]
~~I have just created the channel `##homebrew-pkg` on Libera.chat! The `##` prefix means that the channel is _not_ officially affiliated with the Homebrew project. I encourage people to join either directly via IRC on Libera, or through the Matrix.org bridge.~~

Edit: never mind! The `#machomebrew` channel was migrated from Freenode to Libera, it has ~40 users right now.
